libc/src/__support/FPUtil/aarch64/fenv_darwin_impl.h

```diff
@@ -17,10 +17,10 @@
 #endif
 
 #include <arm_acle.h>
-#include <fenv.h>
 #include <stdint.h>
 
 #include "src/__support/FPUtil/FPBits.h"
+#include <fenv.h>
```

Comment:
Keep #include <fenv.h> in where it was, and add #include "hdr/fenv_macros.h here also. It seems redundant, but it will make the next step easier.

Reply:
same

Reply:
actually, most files remove the include of `fenv.h` entirely. Any idea what we're keeping it around for here?

Reply:
intue said that this will make the next step easier to do. I think in the next PR I am going to make a proxy header for the fenv types so maybe thats why.

Reply:
Ok, the point about not moving the include still stands though. Just undo that change please.